<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeekCalendar</class>
 <widget class="QWidget" name="WeekCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>周视图</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="date_1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="date_6">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextBrowser" name="textEvents_1"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="date_3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="date_4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QTextBrowser" name="textEvents_5"/>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="lastWeekButton">
       <property name="text">
        <string>上一周</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weekButton">
       <property name="text">
        <string>本周</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextWeekButton">
       <property name="text">
        <string>下一周</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="date_2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="date_5">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item row="13" column="1">
    <widget class="QTextBrowser" name="textEvents_7"/>
   </item>
   <item row="7" column="1">
    <widget class="QTextBrowser" name="textEvents_3"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="date_7">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextBrowser" name="textEvents_2"/>
   </item>
   <item row="11" column="1">
    <widget class="QTextBrowser" name="textEvents_6"/>
   </item>
   <item row="8" column="1">
    <widget class="QTextBrowser" name="textEvents_4"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dateTitle">
     <property name="text">
      <string>日期</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="dateEvents">
     <property name="text">
      <string>事件</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>日历显示</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
